components/kyma-environment-broker/internal/vmauthoverrides/mock/client_fake.go

```diff
@@ -0,0 +1,98 @@
+package mocks
```

Comment:
This mock code can be automatically be generated with e.g. `mockery`, it is widely used in KEB. See e.g. [this](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/common/orchestration/resolver.go#L20-L23) as an example